ix-dev/stable/wg-easy/app.yaml

```diff
@@ -0,0 +1,39 @@
+app_version: '13'
+capabilities:
+- description: WG Easy is able to use raw sockets.
+  name: NET_RAW
+- description: WG Easy is able to load kernel modules.
+  name: SYS_MODULE
+- description: WG Easy is able to perform various network-related operations.
+  name: NET_ADMIN
+categories:
+- networking
+description: WG-Easy is the easiest way to install & manage WireGuard!
+home: https://github.com/WeeJeWel/wg-easy
+host_mounts: []
+icon: https://media.sys.truenas.net/apps/wg-easy/icons/icon.png
+keywords:
+- wireguard
+- network
+- vpn
+lib_version: 1.0.0
+lib_version_hash: f2610d4386f9f2f65b1e69176effe945f2d9cc7b9489e6386de82f1f63986c4e
+maintainers:
+- email: dev@ixsystems.com
+  name: truenas
+  url: https://www.truenas.com/
+name: wg-easy
+run_as_context:
+- description: WG Easy runs as root user.
+  gid: 0
+  group_name: root
+  uid: 0
+  user_name: root
+screenshots:
+- https://media.sys.truenas.net/apps/wg-easy/screenshots/screenshot1.png
+sources:
+- https://github.com/WeeJeWel/wg-easy
+- https://github.com/truenas/charts/tree/master/charts/wg-easy
+title: WG Easy
+train: stable
+version: 1.0.1
```

Comment:
Any particular reason for `1.0.1` here ?
`1.0.0` should work as well

Reply:
Thanks for the catch!